website/src/client/utils/orbit.ts

```diff
@@ -66,14 +66,15 @@ export function useOrbit() {
         return;
       }
 
+      const deeplink = `expo-orbit://snack/${experienceURL.replace('exp://', '')}`;
```

Comment:
I think this might actually break the URL spec. What you'll get is something like this:

```
expo-orbit://snack/u.expo.dev/XXX?snack=YYY&snack-channel=ZZZ...
```

I don't think the `u.expo.dev` is allowed as path name characters.

Reply:
A good example here is Cloudflare, no idea wtf is going on here but it seems I broke their implementation of matching `CNAME` records with this type of URL https://expo-snack.app/u.expo.dev/xxx?snack=yyy

Reply:
That being said, if it works 🤷

Reply:
Ohh I see, we should probably try to stay in the URL spec. What if we used `expo-orbit://snack/?url=https://u.expo.dev/XXX?snack=YYY&snack-channel=ZZZ...` instead?

Reply:
I've updated this PR and Orbit to support the following format `expo-orbit://snack/?url=exp://u.expo.dev/XXX?snack=YYY&snack-channel=ZZZ`